components/brave_wallet/common/btc_like_serializer_stream.cc

```diff
@@ -15,21 +16,18 @@ void BtcLikeSerializerStream::Push8AsLE(uint8_t i) {
 }
 
 void BtcLikeSerializerStream::Push16AsLE(uint16_t i) {
-  i = base::ByteSwapToLE16(i);
-  base::span<uint8_t> data_to_insert(reinterpret_cast<uint8_t*>(&i), sizeof(i));
-  PushBytes(data_to_insert);
+  PushBytes(base::byte_span_from_ref(
+      base::numerics::U16FromLittleEndian(base::byte_span_from_ref(i))));
```

Comment:
ByteSwapToLE16 convert *to* LE and is a noop for LE platforms(btw do we have one supported?)
Now we always covert *from* LE which seem to work but semantically incorrect

Reply:
TIL we actually always on LE platforms https://chromium-review.googlesource.com/c/chromium/src/+/5369340

Let's then that be `PushBytes(base::byte_span_from_ref(i))`